man/systemd.link.xml

```diff
@@ -946,6 +946,16 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>MDI-X=</varname></term>
+        <listitem>
+          <para>Specifies MDI-X mode for the port. Takes a boolean value or special value
+          <literal>auto</literal>. When true, the MDI-X (crossover) mode will be used. When false, the
+          MDI (straight through) mode will be used. When <literal>auto</literal>, the MDI status is
+          automatically detected. Defaults to unset, and the kernel's default will be used.</para>
+        </listitem>
+      </varlistentry>
```

Comment:
so after reading the wikipedia page it appears to me we should normalize this option, i.e.

call it MDI= and then take "mdi" and "mdix" as parameters. i.e. for the same reason the wikipedia page is called "MDI" and not "MDI-X" we should not name our option "MDI-X" either, but just "MDI". 

(might make sense to make this parser a bit flexible and accept the syntaxes "mdi" and "straight" as equivalent, as well as "mdi-x", "midx" and "crossover" as equivalent.)